### ISSUE TITLE
tmsu repair with 500 files fails with "too many SQL variables"

### ISSUE THREAD

**Issue by olhotak** [oniony/TMSU#53]
To reproduce, I run the following script:
```
#!/bin/bash
set -e
mkdir /tmp/test
cd /tmp/test
tmsu init
for i in `seq 500`; do touch $i; done
tmsu tag --tags tag *
tmsu --version | head -1
tmsu repair
```

The output I get is:
```
tmsu: creating database at '/tmp/test/.tmsu/db'.
tmsu: New tag 'tag'.
TMSU 0.5.2
tmsu: database query failed: too many SQL variables
```

**Comment by oniony** [comment]
Thanks for reporting this. I shall take a look.

**Comment by 0ion9** [comment]
I suggest upgrading if possible, this works for me. Output I get is:

```
tmsu: creating database at '/tmp/test/.tmsu/db'.
tmsu: new tag 'tag'
TMSU 0.6.0
```

**Comment by oniony** [comment]
Ah thanks @0ion9, must have been something we fixed for 0.6 (I thought the error looked familiar).

**Comment by olhotak** [comment]
I confirm that the issue is fixed for me on the master branch (a4997122fa354fd24d936d22613146ce028ce7e9).

Thanks!
